agents-docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+services:
+  message_queue:
+    build: ./src/message_queue
+    ports:
+      - "8100:8100"
+    environment:
+      - MESSAGE_QUEUE_HOST=message_queue
+      - MESSAGE_QUEUE_PORT=8100
+    networks:
+      - ragapp-network
+
+  control_plane:
+    build: ./src/control_plane
+    ports:
+      - "8000:8000"
+    environment:
+      - CONTROL_PLANE_HOST=control_plane
+      - CONTROL_PLANE_PORT=8000
+      - MESSAGE_QUEUE_URL=http://message_queue:8100
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - CONTROL_PLANE_MODEL=gpt-3.5-turbo
+      - CONTROL_PLANE_ORCHESTRATOR_TYPE=pipeline
+    networks:
+      - ragapp-network
+    depends_on:
+      - message_queue
+
+  travel_agent:
+    build: .
+    ports:
+      - "8001:8001"
+    volumes:
+      - ./config:/app/config
+    environment:
+      - AGENT_HOST=travel_agent
+      - AGENT_PORT=8001
+      - CONTROL_PLANE_URL=http://control_plane:8000
+      - MESSAGE_QUEUE_URL=http://message_queue:8100
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - ENVIRONMENT=dev
+      - AGENT_NAME=travel_agent
+      - AGENT_DESCRIPTION=I can use DuckDuckGo search to find information about a country.
+      - AGENT_PROMPT=Ask me about a country and I will provide you with some information about it.
+      - SYSTEM_PROMPT=You're a helpful travel agent, your task is use DuckDuckGo search to find information about a country. You just need to return the information you find.
```

Comment:
that's confusing, we should only have one prompt

Reply:
For pipeline orchestrator we just need the system_prompt, the others is for agentic orchestrator, but i still let it there :D